(GH-404) Added Norwegian Bokmål translation

AdmiringWorm opened the pull request:
This is not a complete translation (as well as some of the translated entries could definitely be improved)

The following have not been translated:
- LocalSourceViewModel (no need to translate)
- PackageVewModel_StartLoadingFormat (Same format can be used in norwegian)
- SettingsView_SubheaderChoco (no need to translate)
- SettingsView_ToggleElevateByDefault (don't know what the correct translation would be)
- SourcesView_Choco (no need to translate)

fixes #404

RichiCoder1 commented:
/cc @AdmiringWorm @patriksvensson @devlead

gep13 commented:
@asbjornu @khellang could I call on a favour? Can I get a second opinion on the translation that @AdmiringWorm has provided here? The more eyes the better. Thanks!

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Ugyldig versjonsstreng.</value>
+  </data>
+  <data name="LocalSourceView_ButtonExport" xml:space="preserve">
+    <value>Eksporter</value>
```
`Eksportér`?

khellang replied:
Disse går igjen. Er usikker på om det finnes en "rett" versjon her. Ser regelmessing både med og uten aksent.

AdmiringWorm replied:
Viss man skal gå etter bokmålsordboka, så er eksporter uten aksent korrekt.
http://ordbok.uib.no/perl/ordbok.cgi?OPP=+eksporter&ant_bokmaal=5&ant_nynorsk=5&bokmaal=+&ordbok=bokmaal

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Oppdater Alle</value>
+  </data>
+  <data name="LocalSourceView_CheckboxPkgsWithUpdates" xml:space="preserve">
+    <value>Vis Kun Pakker med oppdateringer</value>
```
Usikker på blandingen av store og små bokstaver i denne setningen...

AdmiringWorm replied:
det er jeg å, men brukte den blandningen ettersom det er brukt i den engelske teksten.

khellang replied:
Ja, men blandingen her er ikke konsekvent med den Engelske teksten :smile:

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Forfattere</value>
+  </data>
+  <data name="LocalSourceView_Grid_LatestVersion" xml:space="preserve">
+    <value>Seneste Versjonen</value>
```
`Siste Versjon`?

AdmiringWorm replied:
👍

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Forfattere:</value>
+  </data>
+  <data name="PackageView_ButtonInstall" xml:space="preserve">
+    <value>Installer</value>
```
`Installér`?

khellang replied:
Det samme gjelder vel egentlig `PackageView_ButtonReinstall` og `PackageView_ButtonUninstall`.

AdmiringWorm replied:
her igjen har jeg gått etter bokmålsordboka, hvor installer står uten aksent.
http://ordbok.uib.no/perl/ordbok.cgi?OPP=installer&ant_bokmaal=5&ant_nynorsk=5&bokmaal=+&ordbok=bokmaal

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Galleri</value>
+  </data>
+  <data name="PackageView_LastUpdate" xml:space="preserve">
+    <value>Sist oppdatert</value>
```
`Last update` -> `Siste oppdatering`?

AdmiringWorm replied:
Med tanke på plasseringen, menere jeg det passer bedre med `sist oppdatert`.

![image](https://cloud.githubusercontent.com/assets/1474648/23058703/8b23cfe4-f4f6-11e6-8dcc-c03f05e2aa1b.png)

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Lisens</value>
+  </data>
+  <data name="PackageView_Maintainers" xml:space="preserve">
+    <value> | Vedlikeholdere</value>
```
Mangler `: ` på slutten.

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Pakke ID</value>
+  </data>
+  <data name="PackageView_PackageSize" xml:space="preserve">
+    <value>Pakke størrelse</value>
```
Ser at den engelske versjonen bruker title case på de fleste av disse. Er det en idé å gjøre her også?

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Pakke størrelse</value>
+  </data>
+  <data name="PackageView_ProjectSite" xml:space="preserve">
+    <value>Prosjekt side</value>
```
Ett ord?

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Produktmerknader</value>
+  </data>
+  <data name="PackageView_ReportAbuse" xml:space="preserve">
+    <value>Rapporter misbruk</value>
```
`Rapportér`?

AdmiringWorm replied:
igjen i henhold til bokmålsordboka er det uten aksent.
http://ordbok.uib.no/perl/ordbok.cgi?OPP=rapporter&ant_bokmaal=5&ant_nynorsk=5&bokmaal=+&ordbok=bokmaal

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>nedlastinger</value>
+  </data>
+  <data name="RemoteSourceView_SearchBoxText" xml:space="preserve">
+    <value>Søk</value>
```
Mangler `:`?

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+  </data>
+  <data name="Resources_ThisPC" xml:space="preserve">
+    <value>Denne PC</value>
+    <comment>Potentially change to Denne datamaskinen (this computer)</comment>
```
Tror denne funker 👍

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Chocolatey Gui</value>
+  </data>
+  <data name="SettingsView_Credits" xml:space="preserve">
+    <value>Rulletekst</value>
```
Tror `Kreditering` funker bedre her?

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Rulletekst</value>
+  </data>
+  <data name="SettingsView_Features" xml:space="preserve">
+    <value>Funksjoner</value>
```
`Funksjonalitet`?

AdmiringWorm replied:
Usikker.
Funksjoner høres mer rett ut i hodet mitt.
Meningen er jo at dette skal være en liten overskrift til en liste over funksjoner som du kan skru av å på.

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Verdi</value>
+  </data>
+  <data name="SettingsView_ReleaseNotes" xml:space="preserve">
+    <value>Produktmerknader</value>
```
Alternativer: `Versjonsmerknader`, `Utgivelsesnotater` kanskje `Release Notes` er like greit?

AdmiringWorm replied:
Liker lyden av Utgivelsesnotater.

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>innstillinger</value>
+  </data>
+  <data name="SettingsView_SourcesPath" xml:space="preserve">
+    <value>Bane</value>
```
Eller `Sti`?

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Deaktivert</value>
+  </data>
+  <data name="Controls_PackagesContextMenuInstall" xml:space="preserve">
+    <value>Installer</value>
```
`Installér`?

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Reinstaller</value>
+  </data>
+  <data name="Controls_PackagesContextMenuUninstall" xml:space="preserve">
+    <value>Avinstaller</value>
```
`Avinstallér`?

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+Feil: {2}{3}</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_InstallFailedTitle" xml:space="preserve">
+    <value>Mislyktes i å installere pakke</value>
```
`Installering av pakken feilet.`?

khellang replied:
Hvis du går for denne, så er det sikkert greit å være konsekvent på resten også :smile:

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Mislyktes i å installere pakke</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_PinFailedMessage" xml:space="preserve">
+    <value>Mislyktes i å fest pakken "{0}", versjon "{1}".
```
Fest**e**

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+Feil: {2}{3}</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_PinFailedTitle" xml:space="preserve">
+    <value>Mislyktes i å feste pakke</value>
```
Disse manger `.` på slutten og burde nok være `pakken` i stedet.

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+{0}</value>
+  </data>
+  <data name="RemoteSourceViewModel_FeedSearchError" xml:space="preserve">
+    <value>Feed-søk feil</value>
```
Hmm. Feed. Hva med `Kilde`? `Feil ved kildesøk`?

asbjornu replied:
Enig i at "kilde" er et godt ord her.

asbjornu replied:
Uansett hvordan "feed" oversettes (eller ikke), så er det særskriving i setningen nå, så "Feil ved kanalsøk" e.l. er uansett en forbedring, synes jeg.

AdmiringWorm replied:
Ja det er definitivt bedre.
Særskrivingen var nesten med vilje fordi jeg ikke klarte å tenke på en korrekt oversettelse.

khellang commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Laster inn side {0}...</value>
+  </data>
+  <data name="RemoteSourceViewModel_UnableToConnectToFeed" xml:space="preserve">
+    <value>Kunne ikke koble til feed med kilden: {0}. Vennligst sjekk om denne feeden er tilgjengelig, og prøv igjen.</value>
```
Ah, ser her at det funker dårlig å bruke `kilde` for `feed` 😛

AdmiringWorm replied:
netopp, å siden jeg ikke kunne finne en oversettelse på dette så lot jeg være å oversette `feed`.
Høres dumt ut, men burde i det minste være forståelig.

asbjornu replied:
"Feed" kan kanskje oversettes til "kanal"?

AdmiringWorm replied:
Tusen takk, skjønner ikke hvorfor kanal ikke engang kom til meg.

AdmiringWorm commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="Argument_cant_be_null_or_empty" xml:space="preserve">
+    <value>Argumentet kan ikke være null eller tomt.</value>
```
jeg tror at `kan` er mer korrekt her enn `må`, men er litt usikker der.

AdmiringWorm replied:
Jeg vet ikke hvor dette er brukt, men tror det er brukt for a validere C# argumentet.

AdmiringWorm replied:
@gep13 
Do you know where the string `Argument can't be null or empty.` is used?

gep13 replied:
Off the top of my head, not sure. Would need to dig around. Currently working off 4 hours sleep after being up for 27 hours, so not the best person to ask just now :smile: Might be in a position to check tomorrow.

AdmiringWorm commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,550 @@
+    <value>Argumentet kan ikke være null eller tomt.</value>
+  </data>
+  <data name="ChocolateyDialog_ConsoleOutput" xml:space="preserve">
+    <value>Konsolltext</value>
```
Nærmeste oversettelsen jeg kunne finne på `Console Output`

khellang replied:
Ja, men "text" er ikke spesielt Norsk :smile:

AdmiringWorm replied:
Ops, den så jeg ikke.

AdmiringWorm commented:
@khellang I've updated the translation, could you please have another look?

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value> | Vedlikeholdere: </value>
+  </data>
+  <data name="PackageView_PackageID" xml:space="preserve">
+    <value>Pakke ID</value>
```
"Pakke-ID".

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Pakke ID</value>
+  </data>
+  <data name="PackageView_PackageSize" xml:space="preserve">
+    <value>Pakke Størrelse</value>
```
"Pakkestørrelse" eller "Pakke-størrelse".

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Sammendrag</value>
+  </data>
+  <data name="PackageView_TotalDownloads" xml:space="preserve">
+    <value>Totalt Nedlastet</value>
```
Liten "n" i "nedlastet".

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Siste</value>
+  </data>
+  <data name="RemoteSourceView_ButtonRefreshPkgs" xml:space="preserve">
+    <value>Oppdater Pakkeliste</value>
```
Liten "p" i "pakkeliste".

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Oppdater Pakkeliste</value>
+  </data>
+  <data name="RemoteSourceView_CheckboxAllVersions" xml:space="preserve">
+    <value>Alle Versjoner</value>
```
Liten "v" i "versjoner".

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Alle Versjoner</value>
+  </data>
+  <data name="RemoteSourceView_CheckboxIncludePrerelease" xml:space="preserve">
+    <value>Inkluder Forhåndsversjoner</value>
```
Liten "f" i "forhåndsversjoner".

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Søk:</value>
+  </data>
+  <data name="Resources_ThisPC" xml:space="preserve">
+    <value>Denne Datamaskinen</value>
```
Liten "D" i "datamaskinen"

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Lagre</value>
+  </data>
+  <data name="SettingsView_ChocoGui" xml:space="preserve">
+    <value>Chocolatey Gui</value>
```
Gir det mening å oversette "Gui" her til f.eks. "grensesnitt"? I hvilken sammenheng dukker dette opp?

AdmiringWorm replied:
Med tanke på at programmet heter Chocolatey Gui, så mener jeg at det ikke passer å oversette dette. I denne sammenhengen så dukker Chocolatey Gui opp hvor innstillingene relatert til programmet kan justeres.

![image](https://cloud.githubusercontent.com/assets/1474648/23065749/707e7c2c-f517-11e6-91e3-26e3f6e73f5c.png)

asbjornu replied:
Skjønner. Men jeg ser at det av og til omtales som "ChocolateyGUI" (sammentrukket) også. Er det inkonsistens i Chocolatey sin engelske utgave også?

AdmiringWorm replied:
yup, stemmer det.
Egentlig så kunne jeg jo ha fjernet denne, ettersom den vanlige `Resource.resx` filen hadde blitt brukt for denne strengen istedenfor.

asbjornu replied:
@gep13: Why is "Chocolatey Gui" not written as "ChocolateyGUI" consistently? Is there a difference or is this a "bug" in the English resource texts?

AdmiringWorm replied:
@asbjornu Looks like it's an open issue regarding this actually: https://github.com/chocolatey/ChocolateyGUI/issues/402

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Sti</value>
+  </data>
+  <data name="LocalSourceView_CheckboxMatchExact" xml:space="preserve">
+    <value>Treff på eksakte ord</value>
```
Hva med "Kun nøyaktige treff"?

AdmiringWorm replied:
My bedre, takk

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Treff på eksakte ord</value>
+  </data>
+  <data name="RemoteSourceView_CheckboxMatchExactly" xml:space="preserve">
+    <value>Treff på eksakte ord</value>
```
Hva med "Kun nøyaktige treff"?

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Fest</value>
+  </data>
+  <data name="Controls_PackagesContextMenuReinstall" xml:space="preserve">
+    <value>Reinstaller</value>
```
"Installer på nytt"

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Fest</value>
+  </data>
+  <data name="Bootstrapper_UnhandledException" xml:space="preserve">
+    <value>Ubehandled Unntak</value>
```
"Ubehandlede unntak"

AdmiringWorm replied:
Eller kanskje "Ubehandlet unntak"?
Tror det er meningen at dette skal være tittelen når ett generelt ubehandlet unntakk oppstår

asbjornu replied:
Ja, man må vel vite kontekst for å si noe om det skal være entall eller flertall. "Unhandled" kan jo benyttes i begge tilfeller på engelsk.

AdmiringWorm replied:
Mener den engelske versjonen hadde `Unhandled Exception` hvilket ville si at det er entall.

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Ubehandled Unntak</value>
+  </data>
+  <data name="ChocolateyCustomSchemeProvider_InvalidHtml" xml:space="preserve">
+    <value>&lt;h3&gt;Ugyldig HTML Streng&lt;/h3&gt;</value>
```
"Ugyldig HTML" er vel godt nok? Hvis "streng" må være med, skal den bindes sammen med "HTML" med en bindestrek og skrives med liten "s".

AdmiringWorm replied:
👍

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>&lt;h3&gt;Ugyldig HTML Streng&lt;/h3&gt;</value>
+  </data>
+  <data name="ChocolateyCustomSchemeProvider_UnknownHost" xml:space="preserve">
+    <value>&lt;h1&gt;Ukjent Vert&lt;/h1&gt;</value>
```
"Ukjent vert"

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>&lt;h1&gt;Ukjent Vert&lt;/h1&gt;</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_Escalating" xml:space="preserve">
+    <value>Eskalering</value>
```
Skal det være "Eskalering" eller "Eskalerer"? "Escalating" er vel mer tilnærmet det siste. "Eskalering" er vel "escalation" på engelsk.

AdmiringWorm replied:
Helt rett, skal skifte till Eskalerer. Sto helt stille for meg når jeg oversatte denne.

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Pakker</value>
+  </data>
+  <data name="LocalSourceViewModel_UpdateAvailableForChocolatey" xml:space="preserve">
+    <value>Det er en oppdatering tilgjengelig for chocolatey.</value>
```
Er ikke Chocolatey et egennavn som således bør skrives med stor C?

AdmiringWorm replied:
Det er brukt liten 'c' i den engelske varianten her, derfor bruk av liten 'c' i the norske burde nå vel være rett mener jeg.

asbjornu replied:
Jeg vil nesten si at da er det en bug i den engelske oversettelsen.

@gep13: Why is "chocolatey" written with a lower-case "c" here?

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Pakke - {0}</value>
+  </data>
+  <data name="PackageViewModel_FailedToInstall" xml:space="preserve">
+    <value>Mislyktes i å Installere</value>
```
Liten "i" i "installere".

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Mislyktes i å Installere</value>
+  </data>
+  <data name="PackageViewModel_FailedToPin" xml:space="preserve">
+    <value>Mislyktes i å Feste</value>
```
Liten "F" i "feste".

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Mislyktes i å Feste</value>
+  </data>
+  <data name="PackageViewModel_FailedToReinstall" xml:space="preserve">
+    <value>Mislyktes i å Reinstallere</value>
```
"Mislyktes i å installere på nytt"

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Mislyktes i å Reinstallere</value>
+  </data>
+  <data name="PackageViewModel_FailedToUninstall" xml:space="preserve">
+    <value>Mislyktes i å Avinstallere</value>
```
Liten "a" i "avinstallere".

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Mislyktes i å Avinstallere</value>
+  </data>
+  <data name="PackageViewModel_FailedToUnpin" xml:space="preserve">
+    <value>Mislyktes i å Løsne</value>
```
Liten "l" i "løsne".

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Mislyktes i å Løsne</value>
+  </data>
+  <data name="PackageViewModel_FailedToUpdate" xml:space="preserve">
+    <value>Mislyktes i å Oppdatere</value>
```
Liten "o" i "oppdatere"

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+{1}</value>
+  </data>
+  <data name="PackageViewModel_RanIntoReinstallError" xml:space="preserve">
+    <value>Det oppsto en feil under reinstalleringen av {0}.
```
`Det oppsto en feil da {0} ble forsøkt installert på nytt` e.l.

AdmiringWorm replied:
Tror jeg går får en annen variant:
`Det oppsto en feil under førsøket på å installere {0} på ny.`
Hva tror du?

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Kunne ikke laste inn</value>
+  </data>
+  <data name="RemoteSourceViewModel_FailedToLoadRemotePackages" xml:space="preserve">
+    <value>Kunne ikke laste inn eksterne pakker!
```
Hvorfor utropstegn her, men ikke i alle feilmeldinger?

AdmiringWorm replied:
Fordi utropstegn er brukt i den engelske varianten her, men ikke på de andre.

asbjornu replied:
Merkelig.

@gep13: Why is question mark used to terminate this error message and a few others, but not all?

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+{0}</value>
+  </data>
+  <data name="SettingsViewModel_FeatureUpdatesError" xml:space="preserve">
+    <value>Funksjonsoppdatering Feilet</value>
```
Liten "f" i "feilet".

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Fjerner kilde...</value>
+  </data>
+  <data name="SettingsViewModel_SavingSource" xml:space="preserve">
+    <value>Lagrer Kilde</value>
```
Liten "k" i "kilde".

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Lagrer kilde...</value>
+  </data>
+  <data name="SettingsViewModel_SourcesMissingValue" xml:space="preserve">
+    <value>Kilde må ha en verdi!</value>
```
Hvorfor utropstegn her, men ikke i alle feilmeldinger?

AdmiringWorm replied:
Fordi utropstegn er brukt i den engelske varianten her, men ikke på de andre.

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Kilde må ha en verdi!</value>
+  </data>
+  <data name="SettingsViewModel_SourcesMissingId" xml:space="preserve">
+    <value>Kilde må ha en ID!</value>
```
Hvorfor utropstegn her, men ikke i alle feilmeldinger?

AdmiringWorm replied:
Fordi utropstegn er brukt i den engelske varianten her, men ikke på de andre.

asbjornu commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Kilde - {0}</value>
+  </data>
+  <data name="TypeMustBeASemanticVersion" xml:space="preserve">
+    <value>Linjetype må være en semantisk versjon.</value>
```
Hva med `Linjetype må være en gyldig semantisk versjon.`?

AdmiringWorm replied:
👍

AdmiringWorm commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Oppdater Pakkeliste</value>
+  </data>
+  <data name="LocalSourceView_ButtonUpdateAll" xml:space="preserve">
+    <value>Oppdater Alle</value>
```
sant nok, det er sjeldent man driver med 'Title Case' på norks, derimot er ofte jeg har sett det bli brukt når noe oversettes ifra engelsk til norsk. Hvilken er grunnen til at jeg også har gjort det her.

AdmiringWorm commented on Source/ChocolateyGui/Properties/Resources.nb.resx:
```diff
@@ -0,0 +1,553 @@
+    <value>Installer</value>
+  </data>
+  <data name="PackageView_ButtonReinstall" xml:space="preserve">
+    <value>Reinstaller</value>
```
Nei, det er vel egentlig ikke ett norsk ord. Men ett ord jeg har sett my brukt uansett.
'Installere på nytt' blir vel kanskje litt for langt med tanke på at teksten er på en knapp, alternativet kunne kanskje være å bruke Ominstaller?

gep13 commented:
@AdmiringWorm @khellang @asbjornu just wanted to say thank you to all of you for taking the time go through this translation. Really appreciate it!

asbjornu commented:
@gep13: No problem! Just get yourself some sleep and respond to our questions and I think we'll be good to go with this PR. 💤  😄

AdmiringWorm commented:
Well, almost good to go with the PR. Still have a few more fixes to add that @asbjornu thought of.

AdmiringWorm commented:
@asbjornu Jeg oppdaterte med dine anbefalinger, samt fikset `Title Case`.
Viss du kunne sett over det en gang til så hadde det vært fint.
Viss du og vet om en god oversettelse av `Elevate By Default`, så hadde det vært perfekt.

asbjornu commented:
@AdmiringWorm: Knall. 👍  Jeg synes det ser bra ut nå. I hvilken sammenheng brukes "Elevate By Default"?

AdmiringWorm commented:
@asbjornu Det er en av innstillingene til Chocolatey GUI, den bestemmer om Chocolatey GUI skal starte i administrativ modus som standard eller ikke.
Eneste oversettelsen jeg kunne tenke på var `Hev som standard`, men det hørtes ikke rett ut.

asbjornu commented:
@AdmiringWorm: Hva med "Administrator-tilgang som standard" eller "Administrator-rettigheter som standard"?

AdmiringWorm commented:
hmm, jeg syns det kanskje blir litt for langt med tanke på layouten til Chocolatey GUI.
Men det er kanskje bare meg

![image](https://cloud.githubusercontent.com/assets/1474648/23069321/406e33c8-f527-11e6-8292-98324a4fd9e4.png)

AdmiringWorm commented:
jeg tror jeg går for det uansett.

AdmiringWorm commented:
@gep13 I believe this is ready now (with the exception of On and Off switches, since they aren't translatable yet).

Thanks alot for your help @khellang and @asbjornu

asbjornu commented:
Enig i at "Administrator-rettigheter som standard" ikke ser idéelt. ut. Hva med "Administrativ som standard"?

AdmiringWorm commented:
@asbjornu prøvde det å, så ikke helt idéelt ut det heller. Syntes faktisk at "Administrator-rettighet som standard" hørtes og så bedre ut.

RichiCoder1 commented:
Hmm, I'll look into the on/off switches. I believe they should be translatable. Could you go ahead and add placeholder resource strings for when I do figure it out?

AdmiringWorm commented:
@RichiCoder1 Sure, I'll do that.
Making them translatable should probably be done with a style setter so as not to duplicate any more logic than necessary.
I think the SettingsView already have a style override for the toggleswitch element already.

RichiCoder1 commented:
Yup, they do indeed and that was exactly what I was thinking. We're overriding their style currently to make them look prettier xD

magol commented:
Possibility to translate On/Off has been added with PR #410

Also note that the PR removed some unused string from the resource, and you should also remove the same strings.

AdmiringWorm commented:
@magol thanks for letting me/us know. I won't have access a comp with git until after the weekend. But I'll synchronize the translation with the latest strings and translate any addeded strings next week

mwallner commented:
@AdmiringWorm you're finished with this PR? @gep13 bump

AdmiringWorm commented:
@mwallner no sorry, stuff got in the way. I'll try finish it up today or tomorrow.

AdmiringWorm commented:
@mwallner @gep13 
I believe this is ready now, I've rebased everything toward the latest develop branch, removed obsolete strings and translated the On/Off Switches

AdmiringWorm commented on Source/ChocolateyGui/ChocolateyGui.csproj:
```diff
@@ -492,6 +492,7 @@
       <DependentUpon>Settings.settings</DependentUpon>
       <DesignTimeSharedInput>True</DesignTimeSharedInput>
     </Compile>
+    <EmbeddedResource Include="Properties\Resources.nb.resx" />
```
yeah, only the original Resource file needs it. The only time the culture dependent resource files need it if they need to be referenced directly.

mwallner commented:
hey @AdmiringWorm - could you rebase again towards current develop - I'd like to see this in build 0.15 :-)

AdmiringWorm commented:
@mwallner sure thing, just give me a few minutes and I'll have it ready

AdmiringWorm commented:
@mwallner done

mwallner commented:
Thanks for your contribution, it's great to have Norwegian localization available in the upcoming release!
